src/+io/loadTimeSeries.m

```diff
@@ -122,13 +122,19 @@
         ScopeParameters.SMC = load(fullfile(path_input, SMC_file));
     end
 
-    %% 7. Leaf biochemical parameters
-    if ~isempty(Vcmax_file)
-        Vcmaxtable = load(fullfile(path_input, Vcmax_file));
-        ScopeParameters.Vcmo  = interp1(Vcmaxtable(:, 1), Vcmaxtable(:, 2), t_);
-    else
-        ScopeParameters.Vcmo = leafbio.Vcmo * ones(size(t_));
+    %% 7. Leaf Vcmo, Tparam, m, Type, Rdparam, and leafwidth parameters
+    % where landcoverClass is an vector like ["forest"; "forest"; "forest"; "shrubland"; ...]
+    landcovers = unique(landcoverClass, 'stable');
+    for ii = 1:length(xyt.t)
+        landcoverIndex = find(strcmp(landcoverClass(ii), landcovers));
+        ScopeParameters.Vcmo(ii, 1) = ScopeParameters.lcVcmo(landcoverIndex, 1);
+        ScopeParameters.Tparam(ii, :) = ScopeParameters.lcTparam(landcoverIndex, :);
+        ScopeParameters.m(ii, 1) = ScopeParameters.lcm(landcoverIndex, 1);
+        ScopeParameters.Type(ii, 1) = ScopeParameters.lcType(landcoverIndex, 1);
+        ScopeParameters.Rdparam(ii, 1) = ScopeParameters.lcRdparam(landcoverIndex, 1);
+        ScopeParameters.leafwidth(ii, 1) = ScopeParameters.lcleafwidth(landcoverIndex, 1);
     end
+    ScopeParameters = rmfield(ScopeParameters, {'lcVcmo', 'lcTparam', 'lcm', 'lcType', 'lcRdparam', 'lcleafwidth'});
```

Comment:
Can you add a comment on why the fields are removed from `ScopeParameters`?

Reply:
I added an explanation. The `lc...` parameters were landcover dependent (size of unique(landcovers)). In the loadTimeSeries code they are turned into timeseries of e.g. `vcmo`.